express/blocks/shared/video.js

```diff
@@ -0,0 +1,263 @@
+import { getLibs, toClassName } from '../../scripts/utils.js';
+import { isVideoLink, getAvailableVimeoSubLang } from '../../scripts/utils/embed-videos.js';
+
+const { createTag, getConfig, loadBlock } = await import(`${getLibs()}/utils/utils.js`);
```

Comment:
can you move this file and the columns out of shared actually? 
I don't want any more files under /blocks/shared